crates/rpc-client/src/client.rs

```diff
@@ -177,17 +179,24 @@ impl<T> RpcClientInner<T> {
             is_local,
             id: AtomicU64::new(0),
             poll_interval: if is_local { AtomicU64::new(250) } else { AtomicU64::new(7000) },
+            is_custom_poll_interval: AtomicBool::new(false),
         }
     }
 
-    /// Returns the default poll interval (milliseconds) for the client.
+    /// Returns the poll interval (milliseconds) for the client.
     pub fn poll_interval(&self) -> Duration {
         Duration::from_millis(self.poll_interval.load(Ordering::Relaxed))
     }
 
-    /// Set the poll interval for the client in milliseconds.
+    /// Set custom poll interval for the client in milliseconds.
     pub fn set_poll_interval(&self, poll_interval: u64) {
```

Comment:
while we're at it, I think we should consider changing the arg type to duration

Reply:
changed to duration